pkgs/by-name/pr/prometheus-redfish-exporter/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
```

Comment:
I'm not experienced enough in the go department to say that for sure that this is correct, but this diff looks good to me.